pkg/eval/external_cmd_test.go

```diff
@@ -0,0 +1,41 @@
+package eval
+
+import (
+	"os"
+	"testing"
+)
+
+func TestBuiltinFnExternal(t *testing.T) {
+	tmpHome, cleanup := InTempHome()
+	defer cleanup()
+
+	mustMkdirAll("dir")
+	mustCreateEmpty("cmdx")
+	mustWriteFile("cmd1", []byte("#!/bin/sh"), 0755)
+	mustWriteFile("cmd2", []byte("#!/bin/sh"), 0755)
+	mustWriteFile("cmd3", []byte(""), 0755)
+	mustCreateEmpty("file")
+
+	os.Setenv("PATH", tmpHome+":"+os.Getenv("PATH"))
```

Comment:
None of these setups seem to be actually used.

Reply:
Hmmm, I must have inadvertently deleted the tests that use those files. The "cmdx" file, for example, is meant to be used to verify that it is not a usable external command due to the lack of execute permission. But looking at it now it would be better to use `mustWriteFile` to make that explicit -- in addition to adding the test that uses that file.

Reply:
Aha! Those are all from when these tests were combined with the tests in *pkg/eval/external_cmd_unix_test.go*. I had to split the two sets of tests because of MS Windows weirdness. In the process I failed to remove these statements from this unit test. So, basically a copy/paste error.